### PR TITLE
[Metricbeat] Copy k8s.event.message to message field for ECS

### PR DESCRIPTION
**Pull request by ruflin**
The field is not renamed as inside k8s it is used as keyword field and not as text. To keep this the field is copied over.

### CODE REVIEW

File: metricbeat/module/kubernetes/event/_meta/fields.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@
     type: keyword
     description: >
       Message recorded for the given event
+    copy_to: message
```
Comment (webmat) [inline metricbeat/module/kubernetes/event/_meta/fields.yml]: Why `copy_to` and not renaming the field?

Can these messages wrap another message from a container?

Comment (ruflin) [reply metricbeat/module/kubernetes/event/_meta/fields.yml]: The part I was concerned here is that it's keyword and might be also used in other ways. @exekias will know more here.

Comment (webmat) [reply metricbeat/module/kubernetes/event/_meta/fields.yml]: Ah right. Hence why I thought we should make everything `keyword` (and add mf `message.text`) ;-)

But this works

Comment (ruflin) [reply metricbeat/module/kubernetes/event/_meta/fields.yml]: Still think `message` as `text` was a good call ;-)

**Pull request merged**